### PR TITLE
docs(samples): adds queryParamsNamed and queryParamsPositional

### PR DESCRIPTION
**Pull request by steffnay**
Fixes 380

- [x] Tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
 **/node_modules
 src/**/doc/*
 build/
-docs/
```
Comment (JustinBeckwith) [inline .eslintignore]: probably want to keep that

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# How to become a contributor and submit your own code
```
Comment (JustinBeckwith) [inline .github/CONTRIBUTING.md]: *sighs* so running `nodejs-repo-tools` has a side effect of creating/replacing a lot of these files 😟  Can you delete this one?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Thanks for stopping by to let us know something could be better!
```
Comment (JustinBeckwith) [inline .github/ISSUE_TEMPLATE.md]: revert this change

File: .gitignore [hunk 4]
```diff
@@ -11,5 +11,3 @@ system-test/*key.json
 .DS_Store
 google-cloud-logging-winston-*.tgz
 google-cloud-logging-bunyan-*.tgz
-.vscode
```
Comment (JustinBeckwith) [inline .gitignore]: revert this file

File: .jsdoc.js [hunk 5]
```diff
@@ -20,7 +20,7 @@ module.exports = {
   opts: {
     readme: './README.md',
     package: './package.json',
-    template: './node_modules/jsdoc-baseline',
```
Comment (JustinBeckwith) [inline .jsdoc.js]: revert this file

File: .nycrc [hunk 6]
```diff
@@ -1,11 +1,9 @@
 {
```
Comment (JustinBeckwith) [inline .nycrc]: revert this file

File: README.md [hunk 7]
```diff
@@ -78,7 +80,7 @@ More Information: [Google Cloud Platform Launch Stages][launch_stages]
 
 ## Contributing
 
-Contributions welcome! See the [Contributing Guide](https://github.com/googleapis/nodejs-bigquery/blob/master/CONTRIBUTING.md).
+Contributions welcome! See the [Contributing Guide](https://github.com/googleapis/nodejs-bigquery/blob/master/.github/CONTRIBUTING.md).
```
Comment (JustinBeckwith) [inline README.md]: That link probably isn't right.  Just drop the `.github/` part.

File: samples/README.md [hunk 8]
```diff
@@ -17,6 +17,8 @@
   * [Dataset list](#dataset-list)
   * [Tables](#tables)
```
Comment (JustinBeckwith) [inline samples/README.md]: Please `default.profraw` and `package-lock.json`

Comment (JustinBeckwith) [reply samples/README.md]: Also delete the `package-lock.json that's in samples`

File: samples/system-test/.eslintrc.yml [hunk 9]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (JustinBeckwith) [inline samples/system-test/.eslintrc.yml]: please delete this and the directory

File: system-test/.eslintrc.yml [hunk 10]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (JustinBeckwith) [inline system-test/.eslintrc.yml]: please delete this file

**Comment by jkwlui** [comment]
@JustinBeckwith please force merge this. It has 2 links that links to code in samples/ which doesn't exist yet unless merged to `master`:

  [404] https://github.com/googleapis/nodejs-bigquery/blob/master/samples/queryParamsNamed.js
  [404] https://github.com/googleapis/nodejs-bigquery/blob/master/samples/queryParamsPositional.js
